packages/lit-dev-content/site/docs/tools/development.md

```diff
@@ -185,15 +197,11 @@ A few popular options include:
 
 There are a number of IDE plugins that may be useful when developing with Lit. In particular, we recommend using a syntax highlighter that works with Lit templates.
 
-The following plugins highlight Lit templates and check them for errors:
+### lit-plugin
 
-* [`lit-plugin` for VS Code](https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin)
+`lit-plugin` provides syntax highlighting, type checing, and more for Lit templates. It's [available for for VS Code](https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin), or you can use the [`ts-lit-plugin` TypeScript compiler plugin](https://github.com/runem/lit-analyzer/tree/master/packages/ts-lit-plugin) which works with Sublime and Atom.
```

Comment:
`type checing` - minor typo here

Reply:
Double `for` on `available for for VS Code`.